### PR TITLE
Format documentation updates for 6.8.0

### PR DESCRIPTION
**Pull request by dgault**
Adding docs for the 4 below 6.8.0 related items:

- A new reader option for OME-TIFF (ometiff.fail_on_missing_tiff), added in https://github.com/ome/bioformats/pull/3732
- Added  the writer to the DICOM format and a note to acknowledge the work on DICOM WSI (https://github.com/ome/bioformats/pull/3742)
- Added the new noSequential option for bfconvert (https://github.com/ome/bioformats/pull/3742)

### CODE REVIEW

File: src/main/resources/format-pages.txt [hunk 1]
```diff
@@ -1836,6 +1838,20 @@ utilityRating = Fair
 reader = OIRReader
 notes = Support for this format was added in partnership with OLYMPUS EUROPA SE & Co. KG
 
+[Olympus OMP2INFO]
+extensions = .omp2info
+owner = `Olympus <https://www.olympus-global.com>`_
+bsd = no
+software = `Olympus Viewer Plugin for ImageJ <http://imagej.net/OlympusImageJPlugin>`_
+weHave = * several OMP2INFO datasets \n
```
Comment (melissalinkert) [inline src/main/resources/format-pages.txt:1846]: I think the trailing newline here is causing the build to fail.

**Comment by melissalinkert** [comment]
All looks good to me apart from minor issue noted inline.

**Comment by sbesson** [comment]
Changes look good to me. Now that all the library work has been integrated for 5.8.0,  should the release artifacts be built today? THis would allow the documentation pages for the new format to be generated and to evaluate this PR keeping the official announcement target for tomorrow.

**Review by sbesson — APPROVED** [review]

**Review by melissalinkert — APPROVED** [review]

File: src/main/resources/format-pages.txt [hunk 2]
```diff
@@ -604,9 +604,11 @@ opennessRating = Very good
 presenceRating = Good
 utilityRating = Fair
 reader = DicomReader
+writer = DicomWriter
 mif = true
 notes = * DICOM stands for "Digital Imaging and Communication in Medicine". \n
 * Bio-Formats supports both compressed and uncompressed DICOM files. \n
+* Support for reading and writing DICOM whole slide images (DICOM WSI format) was implemented through collaboration with `NCI Imaging Data Commons <https://imaging.datacommons.cancer.gov/>`_  and has been funded in whole or in part with Federal funds from the National Cancer Institute, National Institutes of Health, under Task Order No. HHSN26110071 under Contract No. HHSN2612015000031. \n
```
Comment (sbesson) [inline src/main/resources/format-pages.txt:611]: As shown by https://merge-ci.openmicroscopy.org/jenkins/job/BIOFORMATS-linkcheck/683/ and

```
$ curl -IL https://imaging.datacommons.cancer.gov/
HTTP/1.1 500 Internal Server Error
Date: Thu, 09 Dec 2021 09:12:52 GMT
Content-Type: text/html; charset=utf-8
Content-Length: 12229
Server: gunicorn/19.6.0
Vary: Cookie
X-Frame-Options: SAMEORIGIN
Strict-Transport-Security: max-age=31536000; includeSubDomains; preload
Set-Cookie: csrftoken=NFcBeGjVhSYOVqy2RrhnvTVTamITXA5RSZl7GPTCRqf6sOOSRjqF9V7GCmR8hJSw; expires=Thu, 08 Dec 2022 09:12:52 GMT; Max-Age=31449600; Path=/; SameSite=Lax; Secure
Via: 1.1 google
Alt-Svc: clear
 ```

Should we add to the linkcheck ignore list for now and report upstream?

Comment (sbesson) [reply src/main/resources/format-pages.txt:611]: Possible less direct, I would propose https://datacommons.cancer.gov/repository/imaging-data-commons  as an alternative link here that would return 200

**Comment by sbesson** [comment]
https://merge-ci.openmicroscopy.org/jenkins/job/BIOFORMATS-linkcheck/684/ now green with e121988 included

**Pull request merged**